adaptdl/adaptdl/torch/gradient_noise_scale.py

```diff
@@ -1,34 +1,15 @@
-# Copyright 2020 Petuum, Inc. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-
 import functools
 import logging
-import warnings
-
 import numpy as np
 import torch.distributed
 import torch.optim
+
 from torch.autograd import Variable
-from types import MethodType
 
 logging.basicConfig(level=logging.INFO)
 LOG = logging.getLogger(__name__)
 LOG.setLevel(logging.INFO)
 
-__all__ = ["AdaScale"]
```

Comment:
Change the `__all__` binding to `["GradientNoiseScale"]` rather than remove it so that the logging, `_average_groups`, etc aren't exposed.